### ISSUE TITLE
decreased dendritic branching

### ISSUE THREAD

**Issue by sarahstofel** [mgijax/mammalian-phenotype-ontology#3428]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

decreased dendritic branching

>2. Synonyms (e.g., absent spleen)

decreased dendritic aborization, decreased dendritic aborisation 

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

reduction in the functional process by which neurons form new dendritic trees and branches to create new synapses

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

https://www.sciencedirect.com/science/article/pii/S0091305796004200?via%3Dihub

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

abnormal dendrite physiology 

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

https://orcid.org/0000-0003-2307-1226

**Comment by anna-anagnostop** [comment]
Per GO:0140059,  **dendrite arborization** is defined as: "the process in which the anatomical structures of a dendritic tree are generated and organized into dendritic branches". In GO, dendrite arborization is a child of dendrite morphogenesis GO:0048813 (which is a Biological Process).

The MP generally places branching morphogenesis terms under a relevant development parent term in the **morphology** branch of the hierarchy. Hence, the majority of dendritic branching defects are currently annotated to abnormal dendrite morphology MP:0008143 (or child terms thereof) typically as a decreased number of dendrite branches.

We should create abnormal, increased as well as decreased dendritic branching terms under abnormal dendrite morphology MP:0008143 and review our annotations to see if we need more specific, child branching terms for Purkinje cell dendrites, motor neuron dendrites, retinal ganglion cell dendrites etc.

**Comment by sbello** [comment]
Should this be branch number rather than branching? That is are we looking at the arborization process or just the number of branches? A decrease in branch number could potentially be due to loss or abnormal pruning rather than impaired branching process.

**Comment by sbello** [comment]
How does this relate to the existing term "decreased dendritic spine density" (MP:0020508) definition is "decrease in the number of dendritic spines in a given cross-sectional area of a dendrite"

**Comment by sarahstofel** [comment]
This may not be an applicable difference, but the spine density refers to the number of spines per certain amount of dendrite whereas the spine number refers to amount of spines as a whole on the dendrite.

**Comment by sbello** [comment]
I'll get this in on Monday and will make the density term a child.

**Comment by sbello** [comment]
Added
abnormal - MP:0021017
decreased - MP:0021018
increased - MP:0021019

Left density as a sibling to number terms. I think the logic for this on other terms was that you could have the same number of X but spread over a different area so that density was changed but number was not.

Note, we do not have logical definitions for the density terms. There does not appear to be a PATO term for this meaning of density. I suspect that if PATO added a term for density = count/amount per unit area that this would fall under the term amount.

There is a PATO ticket (https://github.com/pato-ontology/pato/issues/173) for cell density which is related but more specific but we never did resolve this. Probably worth revisiting on a uPheno call.
